### PR TITLE
Email, Sms and Mms channels implementation

### PR DESCRIPTION
**Pull request by martinarapovic**

### CODE REVIEW

File: src/Infobip.Api.SDK/Email/EmailEndpoints.cs [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+
+            var url = "email/1/bulks/status"
+                .AddToQueryString(
+                    new Dictionary<string, object>
+                    {
+                        { "bulkId", bulkId }
+                    }
+                );
+
+            using (var request = new HttpRequestMessage(HttpMethod.Put, url))
```
Comment (dlozina) [inline src/Infobip.Api.SDK/Email/EmailEndpoints.cs:358]: I just took a look at this email endpoint. Creation of `HttpRequestMessage` is something that we could have create to be abstract and generic. I suppose all other channels use same principle. **We will not change it now**. I can do that refactoring assuming all endpoints have functionality. Just making a note.

**Review by dlozina — COMMENTED** [review]
With this pull request did we cover all the channels?

File: src/Infobip.Api.SDK.Tests/Sms/SendSmsMessageTests.cs [hunk 2]
```diff
@@ -0,0 +1,288 @@
+﻿using System;
```
[the remark this answers is not in the document]
Comment (martinarapovic) [reply src/Infobip.Api.SDK.Tests/Sms/SendSmsMessageTests.cs:1]: Sure. I'll add an action for this.

File: src/Infobip.Api.SDK.Tests/Sms/SendTfaPinCodeOverSmsTests.cs [hunk 3]
```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Infobip.Api.SDK.Exceptions;
+using Infobip.Api.SDK.SMS.Models;
+using Xunit;
+
+namespace Infobip.Api.SDK.Tests.Sms
+{
+    public class SendTfaPinCodeOverSmsTests : IClassFixture<MockedHttpClientFixture>
```
[the remark this answers is not in the document]
Comment (martinarapovic) [reply src/Infobip.Api.SDK.Tests/Sms/SendTfaPinCodeOverSmsTests.cs:10]: Ok. I'll add this as a new section to the README.md file.

**Pull request merged**